framework/tools/generators.py

```diff
@@ -14,3 +21,47 @@ def generate_string(length: int, additional_characters: list = None) -> str:
         result += additional_characters
 
     return "".join(result)
+
+
+def generate_user(password: str = DEFAULT_PASSWORD) -> dict:
+    """Generating a user with the specified password
+
+    Args:
+        password: password for user
+    """
+
+    faker = Faker()
+    hashed_password = bcrypt.hashpw(password.encode(), bcrypt.gensalt()).decode()
+    return {
+        "id": faker.uuid4(),
+        "email": faker.email(),
+        "first_name": faker.first_name(),
+        "last_name": faker.last_name(),
+        "password": password,
+        "hashed_password": hashed_password,
+    }
+
+
+def generate_jwt_token(email: str = '', expired: bool = False) -> str:
+    # Payload for the token
+    if not expired:
```

Comment:
the best conditions are to build from a statement, not a contradiction:
```
if expired:
    pass
else:
    pass
```
And may be:
```
 payload = {
    "sub": email,
    "iat": datetime.datetime.utcnow(),
}
if expired:
    payload["exp"] = datetime.datetime.utcnow() - datetime.timedelta(seconds=1)
else:
    payload["exp"] = datetime.datetime.utcnow() + datetime.timedelta(days=1)
```

Reply:
Fixed.